### PR TITLE
Avoid using web3 to fetch blockchain on cowswap and avoid creating orders on pre_checks

### PR DESCRIPTION
**Pull request by ismaelga**

### CODE REVIEW

File: roles_royce/protocols/cowswap/methods_swap.py [hunk 1]
```diff
@@ -1,17 +1,20 @@
+import time
```
Comment (sideris) [inline roles_royce/protocols/cowswap/methods_swap.py:1]: looks unused. Rm please

File: roles_royce/protocols/roles_modifier/contract_methods.py [hunk 2]
```diff
@@ -93,10 +93,9 @@ def get_exec_transaction_with_role_method(
     """Returns the proper version instance of ExecTransactionWithRole (V1 or V2)."""
     version = get_roles_mod_version(role)
     if version == 2:
-        role_method = ExecTransactionWithRoleV2
+        return ExecTransactionWithRoleV2(roles_mod_address, str(role), to, data, operation, value, should_revert)
     else:
```
Comment (sideris) [inline roles_royce/protocols/roles_modifier/contract_methods.py:97]: we can remove the `else` here.

**Review by sideris — CHANGES_REQUESTED** [review]
Minor comments. 

Are we actually, not using web3? I didn't see it being removed here

**Comment by ismaelga** [comment]
@sideris you are right!
I did remove it (first commint) but went back on that because it would be too difficult, but I did other improvements.
Changed the title of the PR

**Review by sideris — APPROVED** [review]

**Pull request merged**